Chat_App/Views/ChatView.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  TopView.swift
+//  chatApp
+//
+//  Created by Ana Dzamelashvili on 4/19/23.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+class ChatView: UIView {
+    let tableView = UITableView()
+    var sentMessages = [Message]()
+    var typingArea = TextView()
+    let context = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
+    
+    
+    private let stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.distribution = .equalSpacing
+        stackView.spacing = Constants.StackView.spacing
+        stackView.backgroundColor = .clear
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return stackView
+    }()
+    
+    
+    
+    //MARK: - Load messages from coreData
+    func loadMessages(with request: NSFetchRequest<Message> = Message.fetchRequest()) {
+        //        commented since we have that argument inside the function
+        //        let request: NSFetchRequest<Item> = Item.fetchRequest()
+        do {
+            sentMessages = try context.fetch(request)
```

Comment:
view shouldn't fetch messages. Create a viewModel or any object that will be responsible for it